station/cli.py

```diff
@@ -168,6 +169,9 @@ def parse_receiver_job(job) -> Tuple[str, datetime.datetime, datetime.datetime]:
 server_config_parser.add_argument("-u", "--url", type=str, help="URL of content server")
 server_config_parser.add_argument("--id", type=str, help="Station ID")
 server_config_parser.add_argument("-s", "--secret", type=hex_bytes, help="HMAC secret shared with content server")
+logging_parser = config_subparsers.add_parser("logging", help="Station logging configuration")
```

Comment:
Now, we have two similar commands: "logs" and "logging". Maybe you should merge them or move to set login level into the "global" config section?

Reply:
Updated the code as discussed on FB. You can now run `station logs` to view the log file or `station logs --level DEBUG|INFO|...` to change the logging level to desired level. There's still `station config logging --level DEBUG|INFO|...` present, because I think it should be possible to change configuration in the `config` scope.

Let me know if you want this removed.

Reply:
"There should be one-- and preferably only one --obvious way to do it."
I think that the best place to put logging level configuration is the config subparser.

Reply:
Removed the `station logs --level X` in b8188edeacffd88365399feee7b67e214e1a9803.